common/web/lm-worker/src/main/model-compositor.ts

```diff
@@ -240,71 +240,77 @@ export default class ModelCompositor {
       //        Whitespace is probably fine, actually.  Less sure about backspace.
 
       let bestCorrectionCost: number;
+      let correctionPredictionMap: Record<string, Distribution<Suggestion>> = {};
+
       const SEARCH_TIMEOUT = this.testMode ? 0 : correction.SearchSpace.DEFAULT_ALLOTTED_CORRECTION_TIME_INTERVAL;
-      for await(let matches of searchSpace.getBestMatches(SEARCH_TIMEOUT)) {
+      for await(let match of searchSpace.getBestMatches(SEARCH_TIMEOUT)) {
         // Corrections obtained:  now to predict from them!
-        let predictionRoots = matches.map(function(match) {
-          let correction = match.matchString;
-
-          // Worth considering:  extend Traversal to allow direct prediction lookups?
-          // let traversal = match.finalTraversal;
+        let correction = match.matchString;
```

Comment:
Can we use `const` for variables that never change? I don't think `correction` is changing?